### PR TITLE
Better: [Customer SLA Indicator] Granularity by Category in Sources API Doc. RD-25403

### PR DESCRIPTION
**Pull request by jmvallet**
Cf. https://jira.ringcentral.com/browse/RD-25403

I updated reference spec (engage-digital_openapi3.yaml) and generated engage-digital_postman2.json with spectrum.

### CODE REVIEW

File: specs/engage-digital_openapi3.yaml [hunk 1]
```diff
@@ -909,12 +908,28 @@ paths:
           schema:
             type: integer
         - description: Customer SLA warning threshold (must be greater than 0 and less than 100, default value is 0).
-
           in: query
           name: sla_threshold
           required: false
           schema:
             type: integer
+        - description: Filtering strategy for Customer SLA indicator
```
Comment (DrissTM) [inline specs/engage-digital_openapi3.yaml:916]:
```suggestion
        - description: Categories filtering strategy for Customer SLA indicator

```

File: specs/engage-digital_openapi3.yaml [hunk 2]
```diff
@@ -8689,9 +8704,16 @@ components:
           type: integer
         sla_threshold:
           description: Customer SLA warning threshold (must be greater than 0 and less than 100, default value is 0).
-
           format: int32
           type: integer
+        sla_filtering_categories_strategy:
+          description: Filtering strategy for Customer SLA indicator ("ignored_categories" or "selected_categories")
```
Comment (DrissTM) [inline specs/engage-digital_openapi3.yaml:8710]:
```suggestion
          description: Categories filtering strategy for Customer SLA indicator ("ignored_categories" or "selected_categories")

```

**Review by DrissTM — CHANGES_REQUESTED** [review]
minor suggestions

File: specs/engage-digital_openapi3.yaml (hunk 1 above)
Comment (jmvallet) [reply specs/engage-digital_openapi3.yaml]: It's updated, thanks

**Review by DrissTM — APPROVED** [review]
LGTM :+1:

**Pull request merged**